docs/guides/operator-private-gcp-confidential-space.md

```diff
@@ -155,7 +161,7 @@ Install Terraform if it is not already installed: visit [terraform.io](https://w
 
 #### Download the Template Files
 
-Follow the instructions you receive when your registration process is complete (see [UID2 Operator Account Setup](#uid2-operator-account-setup)) to download the template files listed in the following table.
+Download the ZIP file listed in Operator Versions. Be sure to select the GCP download for the latest version. Unzip the files to a convenient location. You will have the files listed in the following table.
```

Comment:
The link to Operator Versions is missing. Should be:
Download the ZIP file listed in [Operator Versions](#operator-versions).

But can I suggest:
Download the ZIP file listed in [Operator Versions](#operator-versions) in the GCP Download column.